package.json

```diff
@@ -164,11 +164,18 @@
       ],
       "order": 8
     },
+    "cleanAfterBuild": {
+      "title": "Clean after Build",
+      "description": "Run clean command after every successful build.",
```

Comment:
One option would be to remove the extensions from `cleanPatterns` thereby making `latex:clean` essentially a shallow clean. We could add a `latex:deep-clean` command which then included them. The "Clean After Build" would then be equivalent to `latex:build` -> `latex:clean` not `latex:build` -> `latex:deep-clean`.